app/modules/disk_report/disk_report_model.php

```diff
@@ -72,6 +72,23 @@ public function get_disk_type()
 						".get_machine_group_filter('AND');
         return current($this->query($sql));
     }
+    
+    /**
+     * Get filesystem type statistics
+     * autor tuxudo
+     *
+     **/
+    public function get_volume_type()
+    {
+        $sql = "SELECT COUNT(CASE WHEN VolumeType = 'bootcamp' THEN 1 END) AS bootcamp,
```

Comment:
It may be better/more future proof to do something like ```SELECT VolumeType as type, count(*) as count from diskreport LEFT JOIN reportdata USING (serial_number) WHERE Internal = 1 group by VolumeType```